### PR TITLE
Fix negative plate reader concentration handling

### PR DESCRIPTION
**Pull request by RodolfoSalido**
Fixed read_pico_csv() to limit concentration range to (0 , 60)

### CODE REVIEW

**Comment by antgonza** [comment]
Hey @RodolfoSalido, you need to solve conflicts for this to move forward. If you do `git pull https://github.com/biocore/metagenomics_pooling_notebook master` it should download the latests version of master and show you the conflicts so you can fix them.

**Comment by RodolfoSalido** [comment]
I think I resolved the conflict?

**Comment by antgonza** [comment]
Yup, but now there are flake8 errors, just add a space after each `,`

File: metapool/metapool.py [hunk 1]
```diff
@@ -96,6 +96,9 @@ def read_pico_csv(f, sep='\t', plate_reader='Synergy_HT',
     pico_df[conc_col_name] = \
         pd.to_numeric(pico_df[conc_col_name], errors='coerce')
 
+    # limit concentration range (0 - 60 )
+    pico_df[conc_col_name] = np.clip(pico_df[conc_col_name], 0, 60)
```
Comment (ElDeveloper) [inline metapool/metapool.py:100]: I thought this should only be done for `SpectraMax_i3x` if so can you add something like: `if plate_reader == 'SpectraMax_i3x': ....`

**Review by ElDeveloper — CHANGES_REQUESTED** [review]

**Comment by ElDeveloper** [comment]
@RodolfoSalido also, can you add a test for these changes?

File: metapool/tests/test_metapool.py [hunk 2]
```diff
@@ -190,6 +190,18 @@ def test_read_pico_csv_spectramax(self):
         with self.assertRaises(ValueError):
             read_pico_csv(fp_spectramax, plate_reader='foo')
 
+    def test_read_pico_csv_spectramax_negfix(self):
+        # Tests that Concentration values are clipped
+        # to a range of (0,60), eliminating possible
+        # negative concentration values.
+        fp_spectramax = os.path.join(os.path.dirname(__file__), 'data',
+                                     'pico_spectramax.txt')
```
Comment (antgonza) [inline metapool/tests/test_metapool.py:198]: Thank you @RodolfoSalido; what do you think about adding a small test to make sure that there are actually values out of the range in fp_spectramax? I tried to find them but it was a bit complicated for me.

**Review by antgonza — COMMENTED** [review]
I thought that this PR will have conflicts after merging https://github.com/biocore/metagenomics_pooling_notebook/pull/62 but it seem fine; which is surprising but kind of cool. Anyway, one comment; thank you.

**Comment by RodolfoSalido** [comment]
I can add that test. 

-Rodolfo

**Comment by RodolfoSalido** [comment]
can we merge this ? Need someone to approve my new pico_spectramax.txt file.

**Comment by antgonza** [comment]
Just to be clear, you need to solve the conflicts via CLI in your local version; please email me if you need help with this.

**Comment by RodolfoSalido** [comment]
Not super sure what the new errors are. I managed to fix the issue with the conflict by merging my local branch with the upstream biocore, but now I'm getting new(different) errors.

**Comment by RodolfoSalido** [comment]
Ok, after many small edits, I think I got my local branch to have no conflicts with the base branch, and pass all tests. But I'm getting a check error that I'm not familiar with. Any help?

**Comment by antgonza** [comment]
Thank you @RodolfoSalido; I think the failure was a github actions hiccup that has been solved - I restarted the tests and now are passing.

**Pull request merged**